api/src/middlewares/services/CaseService/CaseService.ts

```diff
@@ -0,0 +1,46 @@
+import { Logger } from "pino";
+import { AlertJob } from "../utils/types";
+import { QueueService } from "../QueueService";
+import { FormField } from "../../../types/FormField";
+import { FormType, PayMetadata } from "../../../types/FormDataBody";
+import { PaymentViewModel, ProcessQueueData, SESSendJob } from "./types";
+import { AnswersHashMap } from "../../../types/AnswersHashMap";
+
+export interface CaseService {
+  logger: Logger;
+
+  queueService: QueueService;
+
```

Comment:
I've removed `templates` as a property to decouple CaseServices from email implementation. (i.e. integrating into orbit would mean templates won't be required)